Common/AR/Ar.md

```diff
@@ -1,97 +1,119 @@
+# Augmented Reality toolkit component
 
-# Augmented Reality (AR) toolkit for ArcGIS Runtime SDK for Qt
-
-The Augmented Reality toolkit provides support for ARKit for iOS and Android.
-This documentation describes the installation and uses of the AR toolkit.
+The Augmented Reality (AR) toolkit component provides support for ARKit for iOS and
+Android apps written using ArcGIS Runtime SDK for Qt. This documentation
```

Comment:
```suggestion
ARCore for Android apps written using ArcGIS Runtime SDK for Qt. This documentation
```

Since we mention ARKit, we should mention ARCore as well.